packages/web-forms/src/components/FormHeader.vue

```diff
@@ -28,4 +97,32 @@ defineProps<{title: string}>();
 		}
 	}
 }
+
+.smaller-screens {
+	// this adds border to parent div so that bottom shadow of immediate child
+	// is visible.
+	border-bottom: 3px solid #fff;
+	
+	// to show ellipsis in the heading
+	// https://css-tricks.com/using-flexbox-and-text-ellipsis-together/
+	min-width: 0;
+
+	> div {
+		box-shadow: 0px 1px 2px 0px #0000004D;
+	}
+
+	h1 {
+		padding-left: 10px;
+		text-overflow: ellipsis;
+		white-space: nowrap;
```

Comment:
We discussed this on the call yesterday, and I don't want to lose the context: if we're truncating text, we might want to think about how to support expanding it. There's potentially a component generalization to consider there.

Adding to that... do we actually want to truncate form title text? In terms of use of space, might it be a better user experience to let it flow, possibly with smaller font size.

Reply:
I have removed the truncation, instead Form title now wraps around.